app/src/main/java/edu/uc/kovaciad/slicetracker/ui/main/SuperFragment.kt

```diff
@@ -0,0 +1,18 @@
+package edu.uc.kovaciad.slicetracker.ui.main
```

Comment:
Creation of new super-class/super-fragment to allow for shared logic and more manageable/concise sub-fragments